tests/integration/test_commands.py

```diff
@@ -1,20 +1,965 @@
+                "tests/data/mock_manifests/Valid_Test_Manifest.csv",
+                "--data_type",
+                "MockComponent",
+                "--project_scope",
+                "syn54126707",
+            ],
+        )
+        assert result.exit_code == 0
+        assert result.output.split("\n")[4] == (
+            "Your manifest has been validated successfully. "
+            "There are no errors in your manifest, "
+            "and it can be submitted without any modifications."
+        )
+
+    def test_validate_invalid_manifest(self, runner: CliRunner) -> None:
+        """Tests for validation with no errors"""
+        result = runner.invoke(
+            model,
+            [
+                "--config",
+                "config_example.yml",
+                "validate",
+                "--manifest_path",
+                "tests/data/mock_manifests/CLI_tests/CLI_patient_invalid.csv",
+                "--data_type",
+                "Patient",
+            ],
+        )
+        assert result.exit_code == 0
+        assert result.output.split("\n")[3] == (
+            "error: For attribute Family History in row 2 it does not appear "
+            "as if you provided a comma delimited string. Please check your entry "
+            "('Random'') and try again."
+        )
+        assert result.output.split("\n")[4].startswith("error: 'Random' is not one of")
+        assert result.output.split("\n")[5].startswith("error: 'Random' is not one of")
+        assert result.output.split("\n")[6].startswith("[['2', 'Family History',")
+
+
+class TestManifestCommand:
+    """Tests the schematic/manifest/commands validate manifest command"""
+
+    def test_generate_empty_csv_manifest(self, runner: CliRunner) -> None:
+        """Generate two empty csv manifests"""
+        result = runner.invoke(manifest, ["--config", "config_example.yml", "get"])
+        assert result.exit_code == 0
+        # Assert the output has file creation messages
+        assert result.output.split("\n")[7] == (
+            "Find the manifest template using this CSV file path: "
+            "tests/data/example.Biospecimen.manifest.csv"
+        )
+        assert result.output.split("\n")[10] == (
+            "Find the manifest template using this CSV file path: "
+            "tests/data/example.Patient.manifest.csv"
+        )
+        # Assert these files were created:
+        assert os.path.isfile("tests/data/example.Biospecimen.manifest.csv")
+        assert os.path.isfile("tests/data/example.Biospecimen.schema.json")
+        assert os.path.isfile("tests/data/example.Patient.manifest.csv")
+        assert os.path.isfile("tests/data/example.Patient.schema.json")
+        # Remove created files:
+        os.remove("tests/data/example.Biospecimen.manifest.csv")
+        os.remove("tests/data/example.Biospecimen.schema.json")
+        os.remove("tests/data/example.Patient.manifest.csv")
+        os.remove("tests/data/example.Patient.schema.json")
```

Comment:
This should exist within a `finally:` block to ensure that an `assert` statement does not prevent the files from being cleaned up